test/test.googleauth.ts

```diff
@@ -1233,7 +1242,8 @@ describe('googleauth', () => {
   it('getCredentials should return error when env const is not set',
      async () => {
        // Set up a mock to return a null path string
-       mockEnvVar('GOOGLE_APPLICATION_CREDENTIALS');
+       blockGoogleApplicationCredentialEnvironmentVariable();
```

Comment:
did this helper already exist for other tests?

Reply:
Yup!